mvg/mvg.py

```diff
@@ -268,10 +268,35 @@ def create_source(self, sid: str, meta: dict):
 
         # Package info to be submitted to db
         source_info = {"source_id": sid, "meta": meta}
-
         self._request("post", "/sources/", json=source_info)
 
-    def list_sources(self):
+    def create_tabular_source(self, sid: str, meta: dict, columns: List[str]):
+        """
+        Creates a tabular source on the server side.
+
+        Parameters
+        ----------
+        sid : str
+            source Id
+
+        meta : dict
+            meta information
+
+        columns : List[str]
+            Data variables
```

Comment:
Mention
- the requirements (timestamp column included and types of remaining). N
- Not possible to update columns later

Reply:
timestamp columns is not required when making the source. It will be added regardless.